packages/components/src/components.d.ts

```diff
@@ -1497,10 +1531,13 @@ declare global {
     'as-legend-color-continuous-point': Components.AsLegendColorContinuousPointAttributes;
     'as-legend-color-continuous-polygon': Components.AsLegendColorContinuousPolygonAttributes;
     'as-legend-color-continuous': Components.AsLegendColorContinuousAttributes;
+    'as-legend-size-bins-line': Components.AsLegendSizeBinsLineAttributes;
     'as-legend-size-bins-point': Components.AsLegendSizeBinsPointAttributes;
     'as-legend-size-bins': Components.AsLegendSizeBinsAttributes;
+    'as-legend-size-continuous-line': Components.AsLegendSizeContinuousLineAttributes;
     'as-legend-size-continuous-point': Components.AsLegendSizeContinuousPointAttributes;
     'as-legend-size-continuous': Components.AsLegendSizeContinuousAttributes;
+    'as-legend-size-line': Components.AsLegendSizeLineAttributes;
```

Comment:
I think we don't need this component

Reply:
That's the real component, the other two are just proxying it right now. I think this is more semantic than naming it 'as-legend-size-continuous' or 'as-legend-size-bins' and then using it on the other one.

If we end up having differences between the two then we can split it into two different ones.